xpresso/applications.py

```diff
@@ -159,7 +159,7 @@ async def __call__(
         send: starlette.types.Send,
     ) -> None:
         scope_type = scope["type"]
-        if scope_type == "http" or scope_type == "websocket":
+        if scope_type in ["http", "websocket"]:
```

Comment:
This one is very on purpose: it is going to be much faster to compare to two items (and if it is `"http"`, which is going to be most of the request, continue after 1 comparison) than it is to always create a 2 element list (allocated on the heap, pretty slow) and then do an `in` (which probably converts the list to an utterable under the hood and stuff, even more overhead)